### DISCUSSION TITLE
我是绳网新用户，先发个帖子试试（

### DISCUSSION THREAD

**Discussion by breeze0317** [share121/inter-knot#1919] [其他]
喂喂，123，321，看的见吗
![原神15](https://github.com/user-attachments/assets/f78368d7-f888-4a94-bef5-8e18bed50e97)

**Comment by Xuanan0** [comment]
**看到**

**Comment by 1451374815** [comment]
看不见

**Comment by yileinafs** [comment]
嗯呢嗯呢！（看的见）
